src/core/datastore/networkstore.spec.js

```diff
@@ -144,6 +144,17 @@ describe('NetworkStore', () => {
   });
 
   describe('findById()', () => {
+    it('should throw an error if an id is not provided', () => {
+      const store = new NetworkStore(collection);
+      return store.findById().toPromise()
+        .then(() => {
+          throw new Error('This should not happen.')
+        })
+        .catch((error) => {
+          expect(error).toBeA(KinveyError);
```

Comment:
To be similar to other tests, should there be another check for error type? Something similar to:
`expect(error.message).toEqual('No id was provided. An id must be provided.');`